--- thin-provisioning-tools-0.6.1/configure.ac.orig	2016-02-10 15:28:49.000000000 +0100
+++ thin-provisioning-tools-0.6.1/configure.ac	2016-04-05 21:39:11.502032549 +0200
@@ -49,10 +49,10 @@
 dnl -- Check for large file support
 AC_SYS_LARGEFILE
 if test x$ac_cv_sys_file_offset_bits = x64; then
-    LFS_FLAGS+="-D_FILE_OFFSET_BITS=64"
+    LFS_FLAGS="$LFS_FLAGS -D_FILE_OFFSET_BITS=64"
 fi
 if test x$ac_cv_sys_large_files = x1; then
-    LFS_FLAGS+="-D_LARGE_FILES"
+    LFS_FLAGS="$LFS_FLAGS -D_LARGE_FILES"
 fi
 
 ################################################################
